src/dataflow/src/sink/kafka.rs

```diff
@@ -325,10 +326,10 @@ where
         input.for_each(|_, rows| {
             rows.swap(&mut vector);
             for ((key, value), time, diff) in vector.drain(..) {
-                let should_emit = if connector.strict {
-                    connector.frontier.less_than(&time)
+                let should_emit = if as_of.strict {
+                    as_of.frontier.less_than(&time)
                 } else {
-                    connector.frontier.less_equal(&time)
+                    as_of.frontier.less_equal(&time)
```

Comment:
I went looking for this code, and am now confused. I would have thought that without `AS OF` the else case should just be true. I guess I'm not clear on what `as_of` captures in the non-strict case, if we don't support `AS OF`.